Clocks/FMI3.xml

```diff
@@ -1,51 +1,43 @@
-  <Clock name="outClock2"         valueReference="1005" causality="output" variability="clock" clockType="communicationPoint" priority="2"/>
-  <Int32 name="inClock1Ticks"     valueReference="2001" causality="output" clockReference="1001"/>
-  <Int32 name="inClock2Ticks"     valueReference="2002" causality="output" clockReference="1002"/>
-  <Int32 name="inClock3Ticks"     valueReference="2003" causality="output" clockReference="1003"/>
-  <Int32 name="totalInClockTicks" valueReference="2004" causality="output" clockReference="1001"/>
-  <Int32 name="result2"           valueReference="2005" causality="output" clockReference="1002"/>
-  <Int32 name="input2"            valueReference="2006" causality="input"  clockReference="1002" start="0" />
-  <Int32 name="output3"           valueReference="2007" causality="output" clockReference="1003"/>
-</ModelVariables>
-
-<ModelStructure>
-  <Output valueReference="1004"/>
-  <Output valueReference="1005"/>
-  <Output valueReference="2001"/>
-  <Output valueReference="2002"/>
-  <Output valueReference="2003"/>
-  <Output valueReference="2004"/>
-  <Output valueReference="2005"/>
-  <Output valueReference="2007"/>
-  <InitialUnknown valueReference="1004"/>
-</ModelStructure>
-
-  <!--
+ <ScheduledExecution modelIdentifier="Clocks"/>
+ <LogCategories>
+  <Category name="logEvents" description="Log events"/>
+  <Category name="logStatusError" description="Log error messages"/>
+ </LogCategories>
+ <DefaultExperiment stopTime="10" stepSize="1"/>
+ <ModelVariables>
+  <Clock name="inClock1" valueReference="1001" causality="input" interval="constant" intervalDecimal="1.0" priority="0"/>
+  <Clock name="inClock2" valueReference="1002" causality="input" interval="triggered" priority="1"/>
+  <Clock name="inClock3" valueReference="1003" causality="input" interval="countdown" priority="2"/>
+  <Clock name="outClock" valueReference="1005" causality="output" interval="triggered" priority="2"/>
+  <Int32 name="inClock1Ticks" valueReference="2001" causality="output"/>
+  <Int32 name="inClock2Ticks" valueReference="2002" causality="output"/>
+  <Int32 name="inClock3Ticks" valueReference="2003" causality="output"/>
+  <Int32 name="totalInClockTicks" valueReference="2004" causality="output"/>
+  <Int32 name="result2" valueReference="2005" causality="output"/>
+  <Int32 name="input2" valueReference="2006" causality="input" start="0"/>
+  <Int32 name="output3" valueReference="2007" causality="output"/>
+ </ModelVariables>
+ <ModelStructure>
+  <Output valueReference="1005" dependencies="1001 1002 1003"/>
+  <Output valueReference="2001" dependencies="1001"/>
+  <Output valueReference="2002" dependencies="1002"/>
+  <Output valueReference="2003" dependencies="1003"/>
+  <Output valueReference="2004" dependencies="1001 1002 1003"/>
+  <Output valueReference="2005" dependencies="1002 2006"/>
+  <Output valueReference="2007" dependencies="1003"/>
+  <Clock valueReference="1001" clockedVariables="1005 2001 2004"/>
```

Comment:
Fine for Alpha.7, but should be changed for alpha.8 or beta.1 due to https://github.com/modelica/fmi-standard/pull/1375